pornography-hosts

```diff
@@ -15636,5 +15636,21 @@
 0.0.0.0 zzbabes.com
 0.0.0.0 zzpornpics.com
 0.0.0.0 zzztube.com
+
+0.0.0.0 ixiporn.com
+0.0.0.0 lustmaza.com
+0.0.0.0 dropmaza.com
+0.0.0.0 artsporn.com
+0.0.0.0 sexfullmovies.com
+0.0.0.0 exmovies.co.in
+0.0.0.0 teensexmix.com
+0.0.0.0 indiansexmms.info
+0.0.0.0 rajwap.me
+0.0.0.0 smutindia.pro
+0.0.0.0 pinkheartmovies.xyz
+0.0.0.0 ggmaal.cyou
+0.0.0.0 kashtanka.tv
+0.0.0.0 lustesthd.com
 0.0.0.0 nsfw.xxx
 0.0.0.0 pinayflix.tv
+0.0.0.0 lustmaza.live
```

Comment:
**NEW**: https://mypdns.org/my-privacy-dns/porn-records/-/issues/1971

Thanks for this one